### PR TITLE
Myweek6

### PR DESCRIPTION
**Pull request by dvontrec**
Created API focused Week6

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,72 +1,107 @@
...
-  selectPokemon = async (name) => {
-    const pokemon = 
-      await fetch(
-        `https://pokeapi.co/api/v2/pokemon/${name}/`,
-        {cache: "force-cache"})
-        
-    const json = await res.json() 
-    this.setState({selectedPokemon: json, search: name})
+  // Event called When the name input is changed
+  onNameChange = (event) => {
```
Comment (ryanplusplus) [inline src/App.js]:
```suggestion
    onNameChange = (event) => {
```

File: src/App.js [hunk 2]
```diff
@@ -1,72 +1,107 @@
...
 
-  selectPokemon = async (name) => {
-    const pokemon = 
-      await fetch(
-        `https://pokeapi.co/api/v2/pokemon/${name}/`,
-        {cache: "force-cache"})
-        
-    const json = await res.json() 
-    this.setState({selectedPokemon: json, search: name})
+  // Event called When the name input is changed
```
Comment (ryanplusplus) [inline src/App.js:54]:
```suggestion
    // Event called When the name input is changed
```

File: src/App.js [hunk 3]
```diff
@@ -1,72 +1,107 @@
...
+  onNameChange = (event) => {
+    // updates the state to match the event target value
+    this.setState({ nameInput: event.target.value });
+
```
Comment (ryanplusplus) [inline src/App.js]:
```suggestion
```

File: src/People.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Person from './Person';
+const People = ({ people, deletePerson }) => {
+  return people.map(function (person) {
+    console.log(person.cell)
+    return (
+      < div className="people" >
```
Comment (ryanplusplus) [inline src/People.js]:
```suggestion
      <div className="people">
```

**Comment by ryanplusplus** [comment]
Just minor formatting nitpicks, looks good otherwise.

**Comment by dvontrec** [comment]
Thanks for catching that!

**Review by ryanplusplus — APPROVED** [review]

**Review by leighPri — APPROVED** [review]

**Pull request merged**